security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -0,0 +1,56 @@
+[id="cert-manager-operator-release-notes"]
+= {cert-manager-operator} release notes
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-release-notes
+
+toc::[]
+
+TODO: Quick overview
+
+:FeatureName: The {cert-manager-operator}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+For more information about the {cert-manager-operator}, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
+
+// TODO: Link to upstream release notes too
+
+// TODO: Confirm 1.7.1 version
+[id="cert-manager-operator-release-notes-1.7.1"]
+== Release notes for {cert-manager-operator} 1.7.1
+
+// TODO: Update if needed
+Issued: 2022-03-29
+
+The following advisory is available for the {cert-manager-operator} 1.7.1:
+
+// TODO: Replace
+* link:https://access.redhat.com/errata/RHEA-TODO[RHEA-TODO - TODO]
+
+[id="cert-manager-operator-1.7.1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* TODO
```

Comment:
@slaskawi Are there any features/enhancements that we can list here in the release notes for this initial release of the cert-manager operator?

Reply:
I'm not sure if we should supply this paragraph for the initial release. If we have to, maybe we should write something very generic like "Initial release"?